website/docs/d/ec2_transit_gateway_vpn_attachment.html.markdown

```diff
@@ -21,16 +21,32 @@ data "aws_ec2_transit_gateway_vpn_attachment" "example" {
 }
 ```
 
+Filter usage:
+
+```hcl
+data "aws_ec2_transit_gateway_vpn_attachment" "test" {
+  filter {
+    name   = "resource-id"
+    values = ["some-resource"]
+  }
+}
+```
+
 ## Argument Reference
 
 The following arguments are supported:
 
-* `transit_gateway_id` - (Required) Identifier of the EC2 Transit Gateway.
-* `vpn_connection_id` - (Required) Identifier of the EC2 VPN Connection.
+* `transit_gateway_id` - (Optional) Identifier of the EC2 Transit Gateway.
+* `vpn_connection_id` - (Optional) Identifier of the EC2 VPN Connection.
+* `filter` - (Optional) One or more name/value pairs to use as filters. There are
+several valid keys, for a full reference, check out
+[describe-transit-gateway-attachments in the AWS CLI reference][1].
```

Comment:
Can you please update this documentation to the longer form configuration block style? e.g. https://github.com/terraform-providers/terraform-provider-aws/pull/12400/files#diff-259bbf21af9ff942736c706b725f4d6dR76-R88 Thank you!